accounts/templates/accounts/project_form.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
These do not need to have their HTML completely defined again.  Look at base.html, create a base.html for the projects app and then make your pages for the projects app extend base.html